test/paper-textarea.html

```diff
@@ -194,39 +194,6 @@
         assert(!input.focused, 'input is blurred');
       });
     });
-
-    suite('a11y', function() {
```

Comment:
If you are deleting all the accessibility tests, please replace them with other tests that test that this is somehow still accessible